components/hdw-spiffs/spiffs_json.c

```diff
@@ -3,7 +3,9 @@
 #include <stdlib.h>
 #include <stdio.h>
 
+#ifndef EMU
 #include "esp_log.h"
```

Comment:
this should exist in the emulator too...

Reply:
You're right.  Not sure why it was having trouble earlier.